dev/devicelab/bin/tasks/module_host_with_custom_build_test.dart

```diff
@@ -0,0 +1,282 @@
+// Copyright (c) 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:flutter_devicelab/framework/apk_utils.dart';
+import 'package:flutter_devicelab/framework/framework.dart';
+import 'package:flutter_devicelab/framework/utils.dart';
+import 'package:path/path.dart' as path;
+
+final String gradlew = Platform.isWindows ? 'gradlew.bat' : 'gradlew';
+final String gradlewExecutable = Platform.isWindows ? gradlew : './$gradlew';
+
+/// Tests that the Android app containing a Flutter module can be built when
+/// it has custom build types and flavors.
+Future<void> main() async {
+  await task(() async {
+
+    section('Find Java');
+
+    final String javaHome = await findJavaHome();
+    if (javaHome == null) {
+      return TaskResult.failure('Could not find Java');
+    }
+
+    print('\nUsing JAVA_HOME=$javaHome');
+
+    section('Create Flutter module project');
+
+    final Directory tempDir = Directory.systemTemp.createTempSync('flutter_module_test.');
+    final Directory projectDir = Directory(path.join(tempDir.path, 'hello'));
+    try {
+      await inDirectory(tempDir, () async {
+        await flutter(
+          'create',
+          options: <String>['--org', 'io.flutter.devicelab', '--template=module', 'hello'],
+        );
+      });
+
+      section('Build ephemeral host app');
```

Comment:
Since this is not essential for this test specifically, can we skip this step to save test time?

Reply:
We do need this to generate the `.android` directory. Perhaps, `flutter create` should create `.android` automatically.

Reply:
we need to run flutter packages get but not necessarily flutter build if I understand correctly.

Reply:
that's true.